src/Swarm/Util/Bench.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+
+module Swarm.Util.Bench where
+
+import Linear (zero)
+import Swarm.Game.CESK (emptyStore, initMachine)
+import Swarm.Game.Display (defaultRobotDisplay)
+import Swarm.Game.Robot
+import Swarm.Language.Context (empty)
+import Swarm.Language.Pipeline (ProcessedTerm)
+import Swarm.Language.Pipeline.QQ (tmQ)
+import System.Clock (fromNanoSecs)
+
+-- | The program of a robot that does nothing.
+idleProgram :: ProcessedTerm
+idleProgram = [tmQ| {} |]
+
+fakeRobot :: Robot
+fakeRobot =
```

Comment:
Why do you need `Util.Benchmark` with an example robot to use in Web API?

Please feel welcome to add stuff to the modules where it belongs. 🙂 They are a bit long, but making new `Util` modules will not help us orient ourselves better in the code.

Reply:
I got rid of the samples for `Robot`, since they're not easily human-readable anyway.  So this change is gone.